### DISCUSSION TITLE
Permision to edit specific fields

### DISCUSSION THREAD

**Discussion by koksikus** [directus/directus#13441] [Archive]
If I set a role with the right to edit only certain fields, they are visible in the collection as editable ones and that's great, but if the condition is not met, these fields are still editable and only after entering the data, when trying to save, a message appears that you are not authorized to do so. Why is this happening? Shouldn't there be an edit lock since it won't be possible to save anyway. Only in the settings of specific fields can be set read-only, but now I have to enter the appropriate conditions in each field, it's a bit weird.
Am i doing something wrong?
Directus 9.11.0

**Comment by azrikahar** [comment]
> but if the condition is not met, these fields are still editable and only after entering the data, when trying to save, a message appears that you are not authorized to do so. 

This sounds like the open issue #11058, so you're not doing something wrong here 👍
